Dockerfile

```diff
@@ -1,11 +1,42 @@
-FROM busybox:ubuntu-14.04
-MAINTAINER Jimmi Dyson <jimmidyson@gmail.com>
+FROM golang:1.9.2-alpine3.6
 
-ADD ./stage/tyk /bin/tyk
-ADD tyk.conf.example /etc/tyk/tyk.conf
-ADD templates/ /etc/tyk/templates/
+RUN apk add --update git vim bash && rm -rf /var/cache/apk/*
 
-EXPOSE 8080
+RUN echo 'alias ll="ls -la"' >> ~/.bashrc
 
-ENTRYPOINT ["/bin/tyk"]
-CMD []
+ARG BUNDLE_GITHUB__COM
+RUN echo -e "machine github.com\n  login ${BUNDLE_GITHUB__COM}" >> ~/.netrc
```

Comment:
why do you need it? I think you need to remove the github bundle. the container does need to access github to run.